src/components/Navbar/index.tsx

```diff
@@ -34,81 +35,52 @@ const Navbar = ({ isAuth }: any) => {
     }
   }
 
+  const isSmallScreen = useMediaQuery('(max-width:819px)')
+
   return (
     <>
-      <Box
-        sx={{
-          flexGrow: 1,
-          width: '100%',
-          overflow: 'hidden'
-        }}
-      >
-        <AppBar
-          position='fixed'
-          sx={{
-            backgroundColor: '#eeeeee',
-            color: '#280606',
-            top: 0,
-            width: '100%'
-          }}
-        >
-          <Toolbar>
+      <Box>
+        <AppBar position='fixed' sx={{ backgroundColor: '#eeeeee', color: '#280606' }}>
+          <Toolbar sx={{ display: 'flex', justifyContent: 'space-between' }}>
             <Typography
               variant='h5'
               component='div'
               sx={{
-                flexGrow: 1,
                 fontWeight: 'bold'
               }}
             >
               Trust Claims
             </Typography>
-            <Box
-              sx={{
-                display: 'flex',
-                justifyContent: 'center',
-                columnGap: 3
-              }}
-            >
+            {isAuth && (
               <>
-                <Paper
-                  component='div'
-                  sx={{
-                    p: '2px 4px',
-                    display: 'flex',
-                    alignItems: 'center',
-                    width: 400
-                  }}
-                >
-                  <InputBase
-                    type='search'
-                    value={searchVal}
-                    placeholder='Search a Claim'
-                    onChange={e => setSearchVal(e.target.value)}
-                    onKeyUp={handleSearchKeypress}
-                    sx={{
-                      ml: 1,
-                      flex: 1
-                    }}
-                  />
-                  <IconButton type='button' sx={{ p: '10px' }} aria-label='search' onClick={handleSearch}>
-                    <SearchIcon />
-                  </IconButton>
-                </Paper>
+                {isSmallScreen && <ProfileDropdown />}
+                {!isSmallScreen && (
+                  <>
+                    <SearchBar />
+                    <ProfileDropdown />
+                  </>
+                )}
```

Comment:
You are using <ProfileDropdown /> regardless of the screen size. Why are you putting it inside the screen logic? Consider this: 
```
<>
  {!isSmallScreen && <SearchBar />}
  <ProfileDropdown />
</>

Reply:
you are right, I changed it